docs/contributing/first-time.md

```diff
@@ -168,15 +197,26 @@ Once you have completed, tested, and linted your code, and created a {ref}`contr
 
 1.  Visit your fork of the Plone repository on GitHub, and [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) against the development branch.
     -   Make your title and description descriptive.
```

Comment:
Suggestion for clarity:

```suggestion
    -   Make your title and description descriptive.
        Remember that the reviewers are looking at many pull requests and need to be able to understand the context quickly.
        A lazy title like "Fixes bug" is meaningless and will get lost.
```

Reply:
I combined and shortened the three suggestions for this section into dc24c323192cda1b89817686b63460bd73cee4d2